scripts/cli/package.json

```diff
@@ -6,64 +6,78 @@
   "bin": {
     "polymesh-cli": "./dist/polymesh_cli.js"
   },
+  "resolutions": {
+    "@polkadot/keyring": "5.5.1",
+    "@polkadot/util-crypto": "5.5.1",
+    "@polkadot/util": "5.5.1"
+  },
   "scripts": {
-    "start": "npm run build",
-    "create": "npm run build && npm run test",
-    "build": "npm run clean:some && tsc -p .",
+    "start": "yarn build",
+    "create": "yarn build && yarn test",
+    "build": "yarn clean:some && tsc -p .",
+    "build:types": "yarn generate:defs && yarn generate:meta",
+    "generate:defs": "ts-node --skip-project node_modules/.bin/polkadot-types-from-defs --package polymesh-typegen/interfaces --input ./src/interfaces",
+    "generate:meta": "ts-node --skip-project node_modules/.bin/polkadot-types-from-chain --package polymesh-typegen/interfaces --endpoint ./alcyone-polymesh.json --output ./src/interfaces",
+    "lint": "tsc --noEmit --pretty",
     "clean:some": "rm -rf ./dist ./docs",
     "clean:all": "rm -rf ./node_modules ./yarn.lock ./dist ./docs",
```

Comment:
What is the reason for removing the lockfile?

Reply:
I added it since I wanted a clean wipe in case of any problems resulting from a module but if adding it is unnecessary or at the very least low probability I can remove it.